workflow/search-filter.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Search & Filter
+---
+
+The Search & Filter features serve as the jumping-off point in your data exploration process within Middleware. Data is made available across all contexts within Middleware, allowing you to understand the interconnectedness of your data.
+
+The Filter Bar and Search Fields are two of the simplest methods for sifting through your data.They are both dynamically generated based on the applications, integrations, and infrastructure you have instrumented within Middleware. These features are inclusive (i.e. union set operations) and will display data that satisfies any of the selected filters.
+
+<Note> For more information on log-based search, navigate to the [Log Monitoring - Search & Filter]() document to learn about our temporary log search solution.  </Note>
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZXJmNDM5d2JmYzd4aTJ2OHg3ZHd3dms2bTJwZ3R1Z3dzcXl0anNkeiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/rEJAUGmdTiJZqvOEBT/giphy.gif" width="750" height="750"/>
```

Comment:
Don't hate me but the infra search is the worst one to use rn. Recommend using APM's search for now as the cleanest to demo.